pvlib/test/test_ivtools.py

```diff
@@ -102,6 +109,31 @@ def test_fit_sdm_cec_sam(get_cec_params_cansol_cs5p_220p):
                 cells_in_series=1, temp_ref=25)
 
 
+@requires_scipy
+def test_fit_sdm_desoto(get_test_specs_params):
+    result = ivtools.fit_sdm_desoto(cells_in_series=60,
+                                    **get_test_specs_params)
+    result_expected = {'I_L_ref': 9.452324509050774,
+                       'I_o_ref': 3.2246097466679494e-10,
+                       'a_ref': 1.5912875522463978,
+                       'R_sh_ref': 125.79869968976132,
+                       'R_s': 0.2978148476600744,
+                       'alpha_sc': 0.005658,
+                       'EgRef': 1.121,
+                       'dEgdT': -0.0002677,
+                       'irrad_ref': 1000,
+                       'temp_ref': 25}
+    del result['optimize_result']
+    assert np.allclose(pd.Series(result), pd.Series(result_expected),
+                       rtol=1e-4)
+    with pytest.raises(RuntimeError) as exc:
```

Comment:
this belongs in its own test such as `test_fit_sdm_desoto_failure`

Reply:
Does this test warrant its own fixture that includes `cells_in_series`?

TBH, I don't see the fixture `test_fit_sdm_desoto` doing much here, because there is only one test. It might be more extensible for the addition of future tests, but in that case I would also put the expected values into the fixture. See [this example](https://github.com/markcampanelli/pvfit/blob/master/pvfit/modeling/single_diode/equation_test.py#L9).

Reply:
I agree that a fixture is not necessary for these tests. The failure test hard codes the values anyways, so it wouldn't even use it.